amazon-kinesis-client/src/main/java/software/amazon/kinesis/lifecycle/NoOpTaskExecutionListener.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Amazon Software License (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/asl/
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+package software.amazon.kinesis.lifecycle;
+
+import software.amazon.kinesis.lifecycle.events.TaskExecutionListenerInput;
+
+/**
+ * A default implementation of TaskExecutionListener that takes no action on task execution.
```

Comment:
Avoid using `default implementation`. 

```java
/**
* NoOp implementation of the TaskExecutionListener, no actions are performed on task execution.
*/
```

Something like this could be better.